test/appliance_add_packages.xml

```diff
@@ -0,0 +1,72 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<!-- Assuming "test_appliance.xml" is in the same directory -->
+<image schemaversion="6.8" name="Ubuntu-22.04_appliance">
+    <inherit path="test/test_appliance.xml"/>
```

Comment:
Usage: `berrymill -i /path/to/appliance.xml`

berrymill will cd in dir "/path/to"

Also with the change I proposed to the Loader.load() method this only works for me with:
`<inherit path="test_appliance.xml"/>`
(Assuming ... is in the same dir)

Reply:
What I mean is:
appliance_add_packages.xml makes the Assumption that test_appliance.xml is in the same dir.
The test_appliance you added is (for testing) most likely the base appliance (I am assuming)

When the User invokes berrymill they need to add the path to the appliance.xml (appliance_add_packages.xml)

The current behavior of berrymill is to change dir into that directory. 
And with that your _traverse() function reads path="test/test_appliance.xml", tries to find that and cant because its really not inside another test dir form that perspective.

Long story short:
I dont know why it works in the unit test but it doesnt work in "production"
we need to think of a coherent working directory structure. 
If you know what I mean